d2graph/d2graph.go

```diff
@@ -1052,6 +1052,34 @@ func (obj *Object) GetDefaultSize(mtexts []*d2target.MText, ruler *textmeasure.R
 	return &dims, nil
 }
 
+func (obj *Object) SizeToContent(contentBox *geo.Box, paddingX, paddingY, desiredWidth, desiredHeight float64, labelDims d2target.TextDimensions) {
```

Comment:
can you leave a comment on how to use this function and its params? just very unclear to me when it's so many params

Reply:
added a comment and simplified the function to use fewer args